src/bgg_companion_api.py

```diff
@@ -108,6 +108,12 @@ def __to_board_game(
                     break
         else:
             name = None
+        cooperative = False
```

Comment:
I'm not sure if this is weird and if it would be better to set cooperative to False in an else statement after the below conditional.  Regardless, I can't without breaking the code due to "calling cooperative before assignment", which I'm not sure why.  With an 
```
else: 
    cooperative = False
```

There is no way cooperative wouldn't be assigned before it's called in the return